tests/ui_tools/test_boxes.py

```diff
@@ -196,6 +196,46 @@ def test_footer_notification_on_invalid_recipients(
             write_box.header_write_box.focus_col == write_box.FOCUS_HEADER_BOX_RECIPIENT
         )
 
+    @pytest.mark.parametrize(
+        "header, expected_recipient_emails, expected_recipient_user_ids",
+        [
+            case(
+                "Human 1 <person1@example.com>",
+                ["person1@example.com"],
+                [11],
+                id="single_recipient",
+            ),
+            case(
+                "Human 1 <person1@example.com>, Human 2 <person2@example.com>",
+                ["person1@example.com", "person2@example.com"],
+                [11, 12],
+                id="multiple_recipients",
+            ),
+            case(
+                "Human 1person1@example.com",
+                ["1person1@example.com"],
+                [],
+                id="invalid_recipient",
+            ),
```

Comment:
I'm removing these test cases, since `self.update_recipients()` will only be called if all the recipients are valid, eliminating the possibility of invalid emails being passed into it.

Reply:
I seemed to have marked the wrong lines here, I've let the "multiple_recipients" case stay, I've removed another case below "invalid_recipient", which was valid and invalid mixed together.